src/content/en/2021/media.md

```diff
@@ -1,20 +1,875 @@
+        <td class="numeric">1.0%</td>
+        <td class="numeric">1.2%</td>
+      </tr>
+      <tr>
+        <td>`(max-width: 400px) 100vw, 400px`</td>
+        <td class="numeric">1.0%</td>
+        <td class="numeric">1.1%</td>
+      </tr>
+      <tr>
+        <td>`(max-width: 800px) 100vw, 800px`</td>
+        <td class="numeric">0.8%</td>
+        <td class="numeric">0.9%</td>
+      </tr>
+      <tr>
+        <td>`(max-width: 500px) 100vw, 500px`</td>
+        <td class="numeric">0.8%</td>
+        <td class="numeric">0.9%</td>
+      </tr>
+      <tr>
+        <td>`(max-width: 1024px) 100vw, 1024px`</td>
+        <td class="numeric">0.7%</td>
+        <td class="numeric">0.9%</td>
+      </tr>
+      <tr>
+        <td>`(max-width: 320px) 100vw, 320px`</td>
+        <td class="numeric">0.5%</td>
+        <td class="numeric">0.8%</td>
+      </tr>
+      <tr>
+        <td>`(max-width: 100px) 100vw, 100px`</td>
+        <td class="numeric">0.7%</td>
+        <td class="numeric">0.8%</td>
+      </tr>
+      <tr>
+        <td>`100vw`</td>
+        <td class="numeric">0.7%</td>
+        <td class="numeric">0.7%</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="A ranked list of the most common sizes attribute values, before Javascript runs.", sheets_gid="228184279", sql_file="image_sizes_attribute_strings.sql") }}</figcaption>
+</figure>
+
+One in ten `sizes` attributes on mobile has an initial value of `auto`. This non-standard value is then presumably replaced by a Javascript library (probably <a hreflang="en" href="https://github.com/aFarkas/lazysizes">lazysizes.js</a>), using the measured layout size of the image.
+
+Some error in `sizes` is acceptable; the attribute provides a pre-layout hint to the browser in order to help it select an appropriate resource to load,_ before_ layout is complete. But large errors can lead to bad resource choices. This appears likely for the least-accurate quarter of `sizes` attributes, which report widths twice as large as the actual `<img>` layout width on desktop and 1.5x as large as the actual `<img>` layout width on mobile.
+
+So: one in ten `sizes` attributes is being authored on the client by a Javascript library, and at least one in four is inaccurate enough that the error is likely to impact resource selection. Both of these facts represent significant opportunities for either <a hreflang="en" href="https://github.com/ausi/respimagelint">existing tooling</a> or <a hreflang="en" href="https://github.com/whatwg/html/issues/4654">new web platform features</a> to help more authors get `sizes` right.
+
+##### `<picture>`
+
+The `<picture>` element serves a couple of use cases:
+
+1. Art direction, with the [`media` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture#the_media_attribute)
+2. Type-switching, via the [`type` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture#the_type_attribute)
```

Comment:
Should this be format switching instead of type switching?

```suggestion
2. Format-switching, based on mime-type, via the [`type` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture#the_type_attribute)
```